job/replay_manager.go

```diff
@@ -120,22 +250,45 @@ func (m *Manager) Close() error {
 }
 
 func (m *Manager) Init() {
+	m.shuttingDownLongRunningReplay()
+
 	logger.I("starting replay workers")
 	for i := 0; i < m.config.NumWorkers; i++ {
 		m.wg.Add(1)
 		go m.spawnServiceWorker()
 	}
 }
 
+func (m *Manager) shuttingDownLongRunningReplay() {
+	replaySpecRepo := m.replaySpecRepoFac.New(models.JobSpec{})
+	runningReplaySpecs, err := replaySpecRepo.GetByStatus(ReplayStatusToValidate)
```

Comment:
renamed the variable to runningReplaySpecs, and now accept ReplayStatusToValidate